### DISCUSSION TITLE
Investigate GFNI for Leopard

### DISCUSSION THREAD

**Discussion by klauspost** [klauspost/reedsolomon#260] [Ideas/Features]
Investigate if GFNI is feasible for leopard encoding.

**Comment by musalbas** [comment]
I think it is because this library also uses the "novel polynomial basis" that leopard uses, with GFNI: https://github.com/firedancer-io/firedancer/blob/bf953b0608342ab5758ae5ef24587012b0c0e072/src/ballet/reedsol/fd_reedsol_gfni_32.S#L18

**Comment by klauspost** [comment]
Now uses GFNI.
